pylint_odoo/misc.py

```diff
@@ -112,14 +113,37 @@ def get_manifest_file(self, node_file):
 
     def set_ext_files(self):
         """Create `self.ext_files` dictionary with {extension_file: [files]}
+            and exclude files using --ignore and --ignore-patterns parameters
         """
         self.ext_files = {}
+        base_path = os.path.dirname(os.path.realpath(__file__))
+        ignores = [os.path.realpath(os.path.join(base_path, ignore))
+                   for ignore in self.linter.option_value('ignore')]
+        ignores_patterns = [glob.glob(os.path.join(base_path, ignore.pattern))
+                            for ignore
+                            in self.linter.option_value('ignore-patterns')]
```

Comment:
I'm not tested this code but I don't understand how works if the `arg.dest` for this parameter is [black_list_re](https://github.com/PyCQA/pylint/blob/da1da56853380a5a387ad287f4398402b14ef123/pylint/lint.py#L278)
Check if It have the same content, please